lib/chef/knife/bootstrap.rb

```diff
@@ -88,7 +88,7 @@ def self.friendly_opt_list(opt_array)
       option :winrm_auth_method,
         short: "-w AUTH-METHOD",
         long: "--winrm-auth-method AUTH-METHOD",
-        description: "The WinRM authentication method to use. Valid choices are #{friendly_opt_list(WINRM_AUTH_PROTOCOL_LIST)}.",
```

Comment:
Why remove the valid values? The only other way to discover them is to try them and see if you get an error.

Reply:
Using in prints them to help as well. It just does so in a kind of confusing way: 

```
-w AUTH-METHOD,                  The WinRM authentication method to use. Valid choices are 'plaintext', 'kerberos', 'ssl' and 'negotiate'. (included in ['plaintext', 'kerberos', 'ssl', 'negotiate'])
        --winrm-auth-method
```

Reply:
Yeah, we should fix that output, but it was weird having them printed twice.

Reply:
:+1:

Reply:
https://github.com/chef/mixlib-cli/pull/59